github-electron/github-electron.d.ts

```diff
@@ -1357,7 +1353,7 @@ declare module GitHubElectron {
 		* Only string properties are send correctly.
 		* Nested objects are not supported.
 		*/
-		extra?: {};
+		extra?: any;
```

Comment:
From document, at least `Object` should be used instead of `any`.

Doc says
> Only string properties are sent correctly.

So `{[prop: string]: string}` may be more suitable to avoid implicit `any`.